webapp/components/Modal/DisableModal.vue

```diff
@@ -1,28 +1,17 @@
 <template>
-  <ds-modal
-    :title="title"
-    :is-open="isOpen"
-    @cancel="cancel"
-  >
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-21T13:06:19Z" title="Tuesday, May 21st 2019, 3:06:19 pm +02:00">May 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -72,8 +72,8 @@ export default {
         this.$toast.success(this.$t('disable.success'))
         this.isOpen = false
         setTimeout(() => {
-          this.$emit('close')
-        }, 1000)
+          this.$emit('close'), location.reload()
```


Please avoid two command on one line. Make it obvious:

```suggestion
          this.$emit('close')
          location.reload()
```